collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -2462,6 +2477,18 @@ static inline void discovery_update_filenames() {
                 }
             }
         }
+
+        // Cgroup directory
+        if (!strncmp(cg->id, "/docker/", 8) && !cg->docker.filename) {
+            snprintfz(filename, FILENAME_MAX, "%s/containers/%s/config.v2.json", docker_directory_base, (cg->id + 8));
+            if (likely(stat(filename, &buf) != -1)) {
+                cg->docker.filename = strdupz(filename);
+                debug(D_CGROUP, "docker filename for cgroup '%s': '%s'", cg->id, cg->docker.filename);
+            } else {
+                debug(D_CGROUP, "docker file for cgroup '%s': '%s' does not exist or netdata does not have permission to access.",
+                      cg->id, filename);
+            }
+        }
```

Comment:
Why do we need this? Should cgroup-name.sh provide labels (it does in k8s)?

Reply:
I was parsing the database directly instead to use script, I already reverted after our meeting.